### DISCUSSION TITLE
profiler ?

### DISCUSSION THREAD

**Discussion by firedtoad** [cloudwu/skynet#1679] [General]
各位大佬有lua profiler  推荐的么，CPU开销有点高，perf看到有一半都在lua里

**Comment by JieTrancender** [comment]
可以看看这个[skynet火焰图](https://www.keyboard-man.com/post/skynet_profile/)

**Reply by sniper00** [reply]
> 可以看看这个[skynet火焰图](https://www.keyboard-man.com/post/skynet_profile/)

看了你的mongo cpu使用率的文章，给你推荐个方案，本地先写binlog, 然后过滤去重写入到mongo

**Reply by JieTrancender** [reply]
> > 可以看看这个[skynet火焰图](https://www.keyboard-man.com/post/skynet_profile/)
> 
> 看了你的mongo cpu使用率的文章，给你推荐个方案，本地先写binlog, 然后过滤去重写入到mongo

谢谢，我研究一下。
